docs/api.rst

```diff
@@ -74,6 +74,29 @@ Shared cache directory
 Platforms
 ~~~~~~~~~
 
+PlatformDirs
```

Comment:
I'd rather just add this class to autodocs and not document it explicitly

Reply:
In that case I'll move its definition after the ABC class. Let me know if that doesn't make sense